diff --git a/node_modules/clappr-youtube-playback/dist/clappr-youtube-playback.js b/node_modules/clappr-youtube-playback/dist/clappr-youtube-playback.js
index 21945f6..1e46895 100644
--- a/node_modules/clappr-youtube-playback/dist/clappr-youtube-playback.js
+++ b/node_modules/clappr-youtube-playback/dist/clappr-youtube-playback.js
@@ -1,13 +1,13 @@
 (function webpackUniversalModuleDefinition(root, factory) {
 	if(typeof exports === 'object' && typeof module === 'object')
-		module.exports = factory(require("Clappr"));
+		module.exports = factory(require("clappr"));
 	else if(typeof define === 'function' && define.amd)
-		define(["Clappr"], factory);
+		define(["clappr"], factory);
 	else if(typeof exports === 'object')
-		exports["YoutubePlayback"] = factory(require("Clappr"));
+		exports["YoutubePlayback"] = factory(require("clappr"));
 	else
 		root["Clappr"] = root["Clappr"] || {};
 		root["Clappr"]["YoutubePlayback"] = factory(root["Clappr"]);
 })(this, function(__WEBPACK_EXTERNAL_MODULE_1__) {
 return /******/ (function(modules) { // webpackBootstrap
 /******/ 	// The module cache
